Source/WebKit/UIProcess/API/APIWebsitePolicies.h

```diff
@@ -112,6 +113,9 @@ class WebsitePolicies final : public API::ObjectImpl<API::Object::Type::WebsiteP
     WebCore::AllowsContentJavaScript allowsContentJavaScript() const { return m_data.allowsContentJavaScript; }
     void setAllowsContentJavaScript(WebCore::AllowsContentJavaScript allows) { m_data.allowsContentJavaScript = allows; }
 
+    WebKit::EnhancedSecurityEnabled enhancedSecurityEnabled() const { return m_enhancedSecurityEnabled; }
+    void setEnhancedSecurityEnabled(WebKit::EnhancedSecurityEnabled enabled) { m_enhancedSecurityEnabled = enabled; }
```

Comment:
I wonder if we should just use `bool` here. Then we'd avoid having two similar enums for Enhanced Security. What do you think?